ovirtlago/utils.py

```diff
@@ -58,6 +58,19 @@ def log_message(self, *args, **kwargs):
     return _BetterHTTPRequestHandler
 
 
+class LagoThreadingTCPServer(ThreadingTCPServer):
+    """
+    A custom multi-threaded TCP server.
+
+    The custom functionality is:
+
+    - Close the threads that serving the requests when the
+      server's thread exits.
+    """
+
+    daemon_threads = True
+
```

Comment:
Do we need to change the default of request_queue_size to something different than the default (5) ?